### DISCUSSION TITLE
2. Woodpecker community call

### DISCUSSION THREAD

**Discussion by 6543** [woodpecker-ci/woodpecker#2146] [General]
# Date

TO Decide Jet
TODO: create https://framadate.org/abc/en/

# [Join](https://public.senfcall.de/woodpecker-ci)
https://public.senfcall.de/woodpecker-ci

# Agenda
TODO: comment so we put it on

- what does work well
- what does not

**Comment by pat-s** [comment]
@6543 Did this actually happen and should this be closed?

**Reply by qwerty287** [reply]
No, it did not happen, nobody planned something here yet
